Main.kt

```diff
@@ -0,0 +1,89 @@
+import java.lang.IllegalArgumentException
+import java.util.Scanner
+
+class Calculator(private var a: Double = 0.0, private var b: Double = 0.0) {
+    fun add() = a + b
+    fun subtract() = a - b
+    fun multiply() = a * b
+    fun divide() = if (b == 0.0) -1.0 else a / b
+}
+
+fun showMenu() {
+    println("-----Choose an option-----")
+    println("1. Add")
+    println("2. Subtract")
+    println("3. Multiply")
+    println("4. Divide")
+    println("5. Exit")
+    print("Enter your choice: ")
+}
+
+fun option() {
```

Comment:
should rename to "showContinueActions()"

Reply:
em thấy mình có thiếu trường hợp nào ở đây không?

Reply:
có phải là em thiếu trường hợp kiểu mình nhập 2 số a b thì chương trình của em chỉ cho chọn một phép tính chứ không thử được các phép tính khác ạ, nếu muốn thế thì lại phải nhập lại từ đầu phép tính ấy ạ?

Reply:
@inusedname  em đã sửa lại rồi ạ